core/src/main/java/io/kestra/plugin/core/trigger/Webhook.java

```diff
@@ -48,6 +48,38 @@
         - 200 if the webhook triggers an execution.
         - 204 if the webhook cannot trigger an execution due to a lack of matching event conditions sent by other application.
 
+        The response body will contain the execution ID if the execution is successfully triggered using the following format:
+        ```json
+        {
+          "tenantId": "your_tenant_id",
+          "namespace": "your_namespace",
+          "flowId": "your_flow_id",
+          "flowRevision": 1,
+          "trigger": {
+            "id": "the_trigger_id",}
```

Comment:
```suggestion
            "id": "the_trigger_id",
```

Not 100% sure but this closing braket is strange.

Reply:
fixed :)